features/page_objects/project_area_selection_page.rb

```diff
@@ -0,0 +1,22 @@
+class ProjectAreaSelectionPage < SitePrism::Page
+
+  section(:user_bar, AdminUserBarSection, AdminUserBarSection::SELECTOR)
+
+  #element(:rmabristolcitycouncil, "#project_area_step_rma_name_bristol_city_council")
+  #element(:psowestofengland, "#project_area_step_rma_name_pso_west_of_england")
```

Comment:
I'd remove commented out lines - no need to keep them cluttering the file up

Reply:
removed in the latest version